hrcase/xml/case/Exiting.xml

```diff
@@ -59,20 +58,22 @@
         </ActivityType>
```

Comment:
The document you linked on the description says the "Schedule Exit Interview" offset should be -10, but you didn't change it (it's 0 now)